src/main/java/com/celements/search/web/IWebSearchService.java

```diff
@@ -17,7 +17,7 @@
 public interface IWebSearchService {
 
   @NotNull
-  public Set<WebSearchPackage> getAvailablePackages(@NotNull DocumentReference configDocRef);
+  public Set<WebSearchPackage> getAvailablePackages(@Nullable DocumentReference configDocRef);
```

Comment:
won't work magically if you don't change the implementation accordingly ;)

Reply:
But if i am a  Wizzard?
Have a look on the implementation, i think it will return. The Catch procedure already add a default set of search packages to the return value.

Reply:
my bad, the wizard Dari can certainly do it!